lib/src/expressions/interpolate/cubic_bezier_interpolation_expression.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:bezier/bezier.dart';
+import 'package:vector_math/vector_math.dart';
```

Comment:
the license pulled in by `vector_math` appears to be both zlib and google https://github.com/google/vector_math.dart/blob/master/LICENSE

the Google license includes the following clause:


> Redistributions in binary form must reproduce the above
copyright notice, this list of conditions and the following disclaimer
in the documentation and/or other materials provided with the
distribution.

I'd rather not bring that in. Are there any alternatives that you could use instead?

Reply:
I wasn't aware this would be an issue. I just double checked and apparently vector_math is already part of the dependency tree on main: 

https://github.com/greensopinion/dart-vector-tile-renderer/blob/02dfe1c9c865fde83c4f74f050fafb5e7fd643b5/pubspec.lock#L336-L342

I could have a look for an alternative if so desired but I am not sure there will be any other suitable packages. The clause itself appears to be a pretty standard one included in BSD-2- and BSD-3-clause licenses which flutter itself is published under as well.